thriftrw/spec/enum.py

```diff
@@ -133,6 +133,13 @@ def from_wire(self, wire_value):
         check.type_code_matches(self, wire_value)
         return wire_value.value
 
+    def _from(self, value):
```

Comment:
If you're adding a new behavior to the TypeSpec contract, can you also add it to the TypeSpec class with a dummy implementation and documentation of what it does?

Also, this appears to be some variation of `to_native`/`from_primitive` but not exactly. If you're just trying to implement validation at `__init__`, this could be a `validate` method. `__init__` and maybe `to_native` can just call the `validate` method. Then when `to_native` is added, it'll just call `__init__` so validation will already be taken care of.